### PR TITLE
background bug

### PR DESCRIPTION
**Pull request by jebeck**
Bugfix for the bug reported [here](https://trello.com/c/oAn5haz7).

TBH, I still don't really understand how this cropped up on staging where tide-whisperer still has the sort (although tide-whisperer seems to be having a lot of problems on staging - getting a lot of 500s and at least one instance of not all data coming back). Fixing what looked fishy in the generation of the data for the background fill fixed it, so `¯\_(ツ)_/¯`

Also had to update some deps to fix the build of the example app (did this recently in [blip too](https://github.com/tidepool-org/blip/commit/2cbbe9f0793009de6c93067595960f84ca4f9cb5#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)), and did all the dev dependencies while I was at it, plus made a change to remove a React deprecation warning.

### CODE REVIEW

**Comment by jh-bate** [comment]
@jebeck yeah I will take a good look at staging tomorrow with regards to the 500's

File: example/components/weekly.js [hunk 1]
```diff
@@ -124,6 +124,7 @@ var Weekly = React.createClass({
     this.setState({
       title: this.getTitle(datetimeLocationEndpoints)
     });
+    console.log(this.refs.chart);
```
Comment (jh-bate) [inline example/components/weekly.js]: did you still want this here?

File: package.json [hunk 2]
```diff
@@ -27,32 +27,26 @@
...
-    "jshint-stylish": "1.0.1",
-    "json-loader": "0.5.1",
-    "jsx-loader": "0.12.2",
-    "less": "2.4.0",
-    "less-loader": "2.1.0",
-    "mocha": "2.2.1",
-    "mversion": "1.10.0",
+    "autoprefixer-loader": "2.0.0",
+    "chai": "3.0.0",
+    "classnames": "^2.1.3",
```
Comment (jh-bate) [inline package.json]: the ^ for the version differs from the rest

Comment (jebeck) [inline package.json]: Thanks, fixed.

File: example/components/weekly.js (hunk 1 above)
Comment (jebeck) [inline example/components/weekly.js]: Ooops, no. Deleted.

**Comment by jh-bate** [comment]
LGTM

**Comment by darinkrauss** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;1ZHQWnew&#x2F;28-platform-return-valid-json-from-tide-whisperer-when-no-data-is-returned"> [platform] Return valid JSON from tide-whisperer when no data is returned</a>

**Pull request merged**